tests/testthat/test-deps_installation_proposal.R

```diff
@@ -162,3 +216,51 @@ test_that("new_min_cohort_deps_installation_proposal correctly handles \">=\" de
 
   test_proposal_common(x, "pkgdepends", "source", "0.2.0", NULL)
 })
+
+test_that("new_min_isolated_deps_installation_proposal correctly handles tern and rtables", {
+  skip_if_offline()
+  skip_if_empty_gh_token()
+
+  d_std_path <- local_description(
+    list(
+      `tern (>= 0.8.3)` = "Import",
+      `rtables (>= 0.6.1)` = "Import",
+      `formatters (>= 0.5.0)` = "Import"
+    ),
```

Comment:
standard strings should work as well

```suggestion
    list(
      "tern (>= 0.8.3)" = "Import",
      "rtables (>= 0.6.1)" = "Import",
      "formatters (>= 0.5.0)" = "Import"
    ),
```